### ISSUE TITLE
ubuntu_bionic nginx.conf contains duplicate ssl settings, shows warnings when starting app

### ISSUE THREAD

**Issue by jvolkenant** [mail-in-a-box/mailinabox#1459]
nginx.conf in bionic adds a SSL Settings section that was not there in 14.04. Should be removed and controlled in conf.d/ssl.conf

```
service nginx status
...snip...
Oct 24 13:33:34 m.example.com systemd[1]: Starting A high performance web server and a reverse proxy server...
Oct 24 13:33:34 m.example.com nginx[16801]: nginx: [warn] duplicate value "TLSv1.2" in /etc/nginx/conf.d/ssl.conf:3
Oct 24 13:33:34 m.example.com nginx[16801]: nginx: [warn] duplicate value "TLSv1.1" in /etc/nginx/conf.d/ssl.conf:3
Oct 24 13:33:34 m.example.com nginx[16801]: nginx: [warn] duplicate value "TLSv1" in /etc/nginx/conf.d/ssl.conf:3
Oct 24 13:33:36 m.example.com nginx[16815]: nginx: [warn] duplicate value "TLSv1.2" in /etc/nginx/conf.d/ssl.conf:3
Oct 24 13:33:36 m.example.com nginx[16815]: nginx: [warn] duplicate value "TLSv1.1" in /etc/nginx/conf.d/ssl.conf:3
Oct 24 13:33:36 m.example.com nginx[16815]: nginx: [warn] duplicate value "TLSv1" in /etc/nginx/conf.d/ssl.conf:3
```

bionic's nginx.conf:
```
        ##
        # SSL Settings
        ##

        ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
        ssl_prefer_server_ciphers on;
```

**Comment by JoshData** [comment]
Good catch. Since they match, we can drop our ssl_protocols line.

**Comment by jvolkenant** [comment]
This fixes it, but does editconf.py support just commenting out lines?
```
sed -i 's/^\tssl_prefer_server_ciphers on;$/\t#ssl_prefer_server_ciphers on;g/' /etc/nginx/nginx.conf
sed -i 's/^\tssl_protocols TLSv1 TLSv1.1 TLSv1.2;/\t#ssl_protocols TLSv1 TLSv1.1 TLSv1.2;/g' /etc/nginx/nginx.conf
```

**Comment by jvolkenant** [comment]
Fixed in #1460
